plugins/auth-backend-module-aws-alb-provider/src/helpers.ts

```diff
@@ -70,6 +70,21 @@ export const makeProfileInfo = (
   };
 };
 
+const getPublicKeyEndpoint = (region: string) => {
+  const commercialEndpoint = `https://public-keys.auth.elb.${encodeURIComponent(
+    region,
+  )}.amazonaws.com`;
+  const govEndpoint = `https://s3-${encodeURIComponent(
```

Comment:
Ah, I meant that we could move down and get rid of these separate declarations too, just returning this straight away

Reply:
Ah sorry! - that's my bad 😅
Will sort it now